pkg/component/etcd/monitoring.go

```diff
@@ -346,12 +347,23 @@ metric_relabel_configs:
 - source_labels: [ __name__ ]
   action: keep
   regex: ^(` + strings.Join(monitoringAllowedMetricsBackupRestore, "|") + `)$`
+
+	monitoringScrapeConfigDruidTmpl = `job_name: ` + monitoringPrometheusJobDruidNamePrefix + `
+honor_timestamps: false
+metrics_path: /federate
+params:
+  'match[]':
+  - '{job="etcddruid",etcd_namespace="{{ .namespace }}"}'
```

Comment:
Use prometheus job name variable here instead of duplicating the job name inline?